README.md

```diff
@@ -1,15 +1,16 @@
 # Adafruit MPR121 Library [![Build Status](https://travis-ci.org/adafruit/Adafruit_MPR121.svg?branch=master)](https://travis-ci.org/adafruit/Adafruit_MPR121)
 
-<img src="https://cdn-shop.adafruit.com/970x728/1982-00.jpg" height="300"/>
+<a href="https://www.adafruit.com/products/1982"><img src="https://cdn-shop.adafruit.com/970x728/1982-00.jpg" height="300"/></a>
 
-This is a library for the Adafruit MPR121 Capacitive touch sensor breakout:
+Tested and works great with the Adafruit MPR121
   * https://www.adafruit.com/products/1982
```

Comment:
please add all the other MPR121 products in adafruit shop :)

Reply:
done, please look is it ok :)